BookingsApi/BookingsApi.DAL/Migrations/20220309154555_AddRemainderNewVenuesForEnglandAndWales.cs

```diff
@@ -0,0 +1,358 @@
+                    { 164, "Folkestone Magistrates Court" },
+                    { 165, "Gateshead Magistrates Court and Family Court" },
+                    { 166, "Gateshead County Court and Family Court" },
+                    { 167, "Gloucester Crown Court" },
+                    { 168, "Gloucester and Cheltenham County and Family Court" },
+                    { 169, "Great Yarmouth Magistrates Court and Family Court" },
+                    { 170, "Great Grimsby Combined Court Centre" },
+                    { 171, "Grimsby Magistrates Court and Family Court" },
+                    { 172, "Guildford Crown Court" },
+                    { 173, "Guildford County Court and Family Court" },
+                    { 174, "Guildford Magistrates Court and Family Court" },
+                    { 175, "Harmondsworth Tribunal Hearing Centre" },
+                    { 176, "Harrogate Justice Centre" },
+                    { 177, "Harrow Crown Court" },
+                    { 178, "Hastings County Court and Family Court" },
+                    { 179, "Hastings Magistrates Court" },
+                    { 180, "Hatfield Magistrates Court" },
+                    { 181, "Haverfordwest County Court and Family Court" },
+                    { 182, "Haverfordwest Magistrates Court" },
+                    { 183, "Hendon Magistrates Court" },
+                    { 184, "Hereford Crown Court" },
+                    { 185, "Hereford Justice Centre" },
+                    { 186, "Hertford County Court and Family Court" },
+                    { 187, "High Wycombe County Court and Family Court" },
+                    { 188, "High Wycombe Magistrates Court and Family Court" },
+                    { 189, "Highbury Corner Magistrates Court" },
+                    { 190, "Horsham County Court and Family Court" },
+                    { 191, "Horsham Magistrates Court" },
+                    { 192, "Hove Trial Centre" },
+                    { 193, "Huddersfield County Court and Family Court" },
+                    { 194, "Hull and Holderness Magistrates Court and Hearing Centre" },
+                    { 195, "Huntingdon Law Courts" },
+                    { 196, "Inner London Crown Court" },
+                    { 197, "Ipswich County Court and Family Hearing Centre" },
+                    { 198, "Ipswich Crown Court" },
+                    { 199, "Ipswich Magistrates Court" },
+                    { 200, "Isle of Wight Combined Court" },
+                    { 201, "Isleworth Crown Court" },
+                    { 202, "Kidderminster Magistrates Court" },
+                    { 203, "King's Lynn Crown Court" },
+                    { 204, "Kingston upon Thames County Court and Family Court" },
+                    { 205, "Kingston upon Thames Crown Court" },
+                    { 206, "Kingston-upon-Hull Combined Court Centre" },
+                    { 207, "Kirklees (Huddersfield) Magistrates Court and Family Court " },
```

Comment:
Does the whitespace at the end of the venue matter?

Reply:
Corrected